internal/sbi/producer/pdu_session.go

```diff
@@ -60,6 +60,10 @@ func HandlePDUSessionSMContextCreate(request models.PostSmContextsRequest) *http
 	smContext.SMLock.Lock()
 	defer smContext.SMLock.Unlock()
 
+	upi := smf_context.GetUserPlaneInformation()
+	upi.Mu.RLock()
+	defer upi.Mu.RUnlock()	
```

Comment:
@tim-ywliu , how can this lint issue be fixed ? Thanks.

Reply:
gofmt -s -w <file>